functional-tests/base-test.py

```diff
@@ -0,0 +1,29 @@
+import os
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
+
+# Setting up saucelabs config
+username = os.environ["SAUCE_USERNAME"]
+access_key = os.environ["SAUCE_ACCESS_KEY"]
+capabilities = {
```

Comment:
We can set up some multi-threading magic here so that we can test all of our supported browsers. 
https://saucelabs.com/blog/selenium-tips-how-to-coordinate-multiple-browsers-in-sauce-ondemand